docs/source/DeveloperGuide/DevelopmentEnvironment/DeveloperInstallation/editingCode.rst

```diff
@@ -46,11 +46,10 @@ on `PyPi <https://pypi.python.org/pypi>`_.
 *git*: DIRAC's version control system of choice is git, so install it.
 
 *basic packages*: you will need at least gcc, python-devel (python all-dev),
-openssl-devel (libssl-dev), mysql-client, libmysqlclient-dev,
-libfreetype6-dev, libncurses5-dev, libjpeg-dev.
+openssl-devel (libssl-dev), mysql-client, libmysqlclient-dev, libfreetype6-dev, libncurses5-dev, libjpeg-dev, libcurl4-gnutls-dev, libgcrypt20-dev, libnghttp2-dev, libpsl-dev, libldap2-dev.
```

Comment:
Woh that's a lot :-) Why do we need all that ? :-) Isn't it all pulled as dependency ?

Reply:
Hello, sorry for my late response. I had to check first before answering. Well for some reason some of these packages are necessary in order to install DIRAC, the package that creates the most issues is pycurl. libcurl4-gnutls-dev and libgcrypt20-dev are necessary and are not pulled as dependencies. I installed DIRAC on my machine 5 times and I had to install them every single time beforehand. As for the other packages, they are necessary too for pycurl but I didn't have to install them in every time. No idea why sometimes they create problems. But it is no harm if they are installed, just to be sure.

Reply:
on which system did you try to install DIRAC ? Which tutorial did you follow ?

Reply:
> But it is no harm if they are installed, just to be sure.

well, probably true, but if we can avoid... ;-)

Reply:
> on which system did you try to install DIRAC ? Which tutorial did you follow ?

It's Ubuntu 18.04 LTS, and I followed the tutorial on this same page that I am trying to add these few changes to.

Reply:
Ah, that's the issue. Ubuntu is not a supported platform. 
Also, see the text just after what you added:
```
The names above are OS-dependent, distribution dependent, and version dependent, so you’ll need to figure it out by yourself how to install them. Some equivalent packages for Fedora/CentOS: python-devel, openssl-devel, mysql, ncurses-libs freetype, libjpeg-devel, MySQL-python. If you are using a OSX machine, then you may end up in more problems than using a linux box.
```
So you may add a new line with the ubuntu equivalent, rather than changing this list. 
This list targets SLC6/CC7, which are the officially supported platforms :-)